src/rules/no-passthrough-getters.ts

```
@@ -0,0 +1,145 @@
+
+      // The nullish coalescing check is already covered by the LogicalExpression check above
+
+      return false;
+    }
+  },
+
+  name: 'no-passthrough-getters',
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'Avoid unnecessary getter methods that simply return properties from constructor parameters',
```

ESLint

[error] Insert `⏎·······`

(prettier/prettier)